### DISCUSSION TITLE
window.ethereum question

### DISCUSSION THREAD

**Discussion by onelove-dev** [Cyfrin/foundry-full-course-cu#1477] [General]
I am  trying to access "window.ethereum" in a browser with MetaMask installed and i am getting undefined, what can i do?

**Comment by EngrPips** [comment]
Hello @manassehO, i would say check manually in your browser console to be sure you have truly installed metamask and confirm that the `ethereum` object is available on the `window` object , you can right click in any modal of your browser and click inspect. then in the console of your browser you should type `window.ethereum` and see if you get an object like i got in the picture below 

<img width="1438" alt="Screenshot 2024-03-13 at 10 07 43 AM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/96abd397-30a4-4b3b-95de-d5ae28b11a23">

And i think you should follow this link https://www.youtube.com/watch?v=pdsYCkUWrgQ (it will take you to Patrick youtube channel) specifically to the tutorial where he was teaching how to connect your front-end to metamask wallet

**Comment by onelove-dev** [comment]
Thanks so much for your solution suggestion . The problem  has been solved.

**Comment by EngrPips** [comment]
glad you resolved it
